CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ option(CORENRN_ENABLE_UNIT_TESTS "Enable unit tests execution" ON)
 option(CORENRN_ENABLE_GPU "Enable GPU support using OpenACC" OFF)
 option(CORENRN_ENABLE_SHARED "Enable shared library build" ON)
 option(CORENRN_ENABLE_LEGACY_UNITS "Enable legacy FARADAY, R, etc" OFF)
+option(CORENRN_ENABLE_NRN_PRCELLSTATE "Enable NRN_PRCELLSTATE debug feature" OFF)
```

Comment:
The option could be `CORENRN_ENABLE_PRCELLSTATE` to avoid any confusion with the `NRN` word?